detekt/wrapper/src/main/java/io/buildfoundation/bazel/detekt/Application.java

```diff
@@ -25,6 +25,7 @@ final class OneShot implements Application {
 
         @Override
         public void run(String[] args) {
+            // TODO: handle custom argument parsing when running as one-shot
```

Comment:
I'm thinking of extracting the pre-processing and post-processing of executing detekt out of WorkerExecutable in order to be able to re-use it here as part of one-shot execution.

Is there a better alternative that you'd recommend?

Reply:
I added an ExecutionUtils class that is used by both `WorkerExecutable` & `OneShot`